kahuna/public/js/search/index.js

```diff
@@ -65,6 +68,20 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                 });
                 return {state: $dsr$.redirect.state, params};
             }]
+        },
+        controllerAs: 'ctrl',
+        controller: ['panels', function(panels) {
+            const ctrl = this;
+            ctrl.collectionsPanel = panels.collectionsPanel;
+            ctrl.metadataPanel = panels.metadataPanel;
+        }],
```

Comment:
Are these needed on top of the similar code in `results.js`? I think they are actually, just checking I understand well.